components/tools/OmeroJava/test/integration/chown/PermissionsTest.java

```diff
@@ -19,11 +19,14 @@
 
 package integration.chown;
 
+import static omero.rtypes.rstring;
```

Comment:
I *think* we try to avoid static imports but if @joshmoore doesn't confirm then don't worry about it.